### ISSUE TITLE
Add a "curators" field to JSON output and change session attribution algorithm

### ISSUE THREAD

**Issue by kimrutherford** [pombase/canto#2594]
We should give attribution to all curators who have made an annotation in a session and export the details of all those users into a new "curators" field in the JSON file

From pombase/website#525:

  - ignore who the session who assigned to
  - attribute those people who have made at least one annotation
  - if any of the annotations are by community curators then call it a community session

Needed for APICURON.

**Comment by jseager7** [comment]
This shouldn't cause any problems for our pipeline, since currently we don't include curator names on our website. I know that we _should_ credit community curators, but I'm pretty sure this use of personal data isn't covered by our privacy policy.

If we decide to credit curators, we might need some way to allow them to opt out of being displayed on the website (ideally when they start their curation session in Canto), even though I think it's unlikely they'd want to curate anonymously. Have you considered this at PomBase?

Currently we exclude the `--all` flag when exporting as JSON so we don't accidentally expose any of the curators' personal data. It would be good if the new `curators` property could also be hidden by default.

**Comment by kimrutherford** [comment]
> This shouldn't cause any problems for our pipeline, since currently we don't include curator names on our website.

OK, thanks.

> If we decide to credit curators, we might need some way to allow them to opt out of being displayed on the website (ideally when they start their curation session in Canto), even though I think it's unlikely they'd want to curate anonymously. Have you considered this at PomBase?

We figure that if they're contributing annotation for their own paper, they won't mind being credited with the annotation.  But in any case we only show the curator names on the publication pages that have the author list (like https://www.pombase.org/reference/PMID:34958661) so we're not displaying anything extra.

If a community curator did want to contribute anonymously they could contact our Helpdesk, but as far as I know that's never happened.

We have a privacy policy in case you're curious: https://www.pombase.org/about/privacy-policy

**Comment by kimrutherford** [comment]
I've added a "annotation_curators" field to the JSON file exported from Canto.  It looks like this:

```json
"annotation_curators" : [
   {
      "annotation_count" : 22,
      "community_curator" : true,
      "name" : "Yasushi Hiraoka"
   },
   {
      "annotation_count" : 12,
      "community_curator" : false,
      "name" : "Valerie Wood"
   }
]
```

It's called "annotation_curators" to make it clear that it contains the curators of the annotations rather than a user who the session was assigned to.  Sometimes the assigned curators do no curation.

I've also added a new flag in the JSON file "has_community_curation" which is set to true if any of the annotations were created by a community curator.
